Kibol-Alert/ViewModels/ClubRelationVM.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Kibol_Alert.ViewModels
+{
+    public class ClubRelationVM
+    {
+        //Work in progress
+    }
+}
```

Comment:
this one could be deleted, because you dont need to have viewmodel of model which will be not used. This model is just for create a table for entity to know, what columns we need